go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/bnb-chain/greenfield-storage-provider
 
-go 1.20
+go 1.19
```

Comment:
Need to update the golang version?

Reply:
It must have been accidentally added; it's now fixed. We're waiting to see if the end-to-end test passes.